Configが読まれない？

```
[09:47:11 ERROR]: [root] 初期化処理に失敗しました。シャットダウンしています…
[09:47:11 WARN]: org.flywaydb.core.internal.exception.FlywaySqlException:
Unable to obtain connection from database (jdbc:mysql://localhost:3306) for user 'root': Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
SQL State  : 08S01
Error Code : 0
Message    : Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.

[09:47:11 WARN]:        at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:60)
[09:47:11 WARN]:        at org.flywaydb.core.internal.database.DatabaseFactory.createDatabase(DatabaseFactory.java:72)
[09:47:11 WARN]:        at org.flywaydb.core.Flyway.execute(Flyway.java:1670)
[09:47:11 WARN]:        at org.flywaydb.core.Flyway.migrate(Flyway.java:1356)
[09:47:11 WARN]:        at com.github.unchama.seichiassist.SeichiAssist.$anonfun$monitoredInitialization$1(SeichiAssist.scala:505)
[09:47:11 WARN]:        at com.github.unchama.util.ClassUtils.withThreadContextClassLoaderAs(ClassUtils.java:16)
[09:47:11 WARN]:        at com.github.unchama.seichiassist.SeichiAssist.monitoredInitialization(SeichiAssist.scala:498)
[09:47:11 WARN]:        at com.github.unchama.seichiassist.SeichiAssist.onEnable(SeichiAssist.scala:657)
[09:47:11 WARN]:        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264)
[09:47:11 WARN]:        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:337)
[09:47:11 WARN]:        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:403)
[09:47:11 WARN]:        at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugin(CraftServer.java:381)
[09:47:11 WARN]:        at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugins(CraftServer.java:330)
[09:47:11 WARN]:        at net.minecraft.server.v1_12_R1.MinecraftServer.t(MinecraftServer.java:422)
[09:47:11 WARN]:        at net.minecraft.server.v1_12_R1.MinecraftServer.l(MinecraftServer.java:383)
[09:47:11 WARN]:        at net.minecraft.server.v1_12_R1.MinecraftServer.a(MinecraftServer.java:338)
[09:47:11 WARN]:        at net.minecraft.server.v1_12_R1.DedicatedServer.init(DedicatedServer.java:272)
[09:47:11 WARN]:        at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:545)
[09:47:11 WARN]:        at java.lang.Thread.run(Thread.java:748)
[09:47:11 WARN]: Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure
```

--- a/build.sbt
+++ b/build.sbt
@@ -9,7 +9,6 @@ ThisBuild / scalaVersion := "2.13.1"
 // ThisBuild / version はGitHub Actionsによって取得/自動更新される。
 // 次の行は ThisBuild / version := "(\d*)" の形式でなければならない。
 ThisBuild / version := "34"
-ThisBuild / name := "SeichiAssist"
 ThisBuild / organization := "click.seichi"
 ThisBuild / description := "ギガンティック☆整地鯖の独自要素を司るプラグイン"
 
@@ -168,6 +167,7 @@ Compile / PB.targets := Seq(scalapb.gen() -> (Compile / sourceManaged).value / "
 // region 各プロジェクトの設定
 
 lazy val root = (project in file(".")).settings(
+  name := "SeichiAssist",
   assembly / assemblyOutputPath := baseDirectory.value / "target" / "build" / s"SeichiAssist.jar",
   libraryDependencies := providedDependencies ++ testDependencies ++ dependenciesToEmbed,
   excludeDependencies := Seq(ExclusionRule(organization = "org.bukkit", name = "bukkit")),

